core/src/main/scala/org/http4s/HttpToken.scala

```diff
@@ -0,0 +1,20 @@
+package org.http4s
+
+import org.http4s.util.{Renderable, Writer}
+
+sealed trait HttpToken extends Renderable
```

Comment:
I count 27 places `token` appears in the HTTP RFCs.  I'm thinking that maybe this is a value rather than a sealed type.

Reply:
I converted `HttpToken` to an `abstract class`. I'm not sure if that is what you have in mind